spec/schedulers/AsapScheduler-spec.ts

```diff
@@ -71,11 +71,11 @@ describe('Scheduler.asap', () => {
     const fakeTimer = sandbox.useFakeTimers();
     // callThrough is missing from the declarations installed by the typings tool in stable
     const stubSetInterval = (<any> sinon.stub(global, 'setInterval')).callThrough();
-    function dispatch(state: any): void {
+    function dispatch(this: any, state: any): void {
```

Comment:
I'd love a second pass that removes some of the `any` types that are being added, but that shouldn't block the PR.